src/EFCore.PG.NTS/Query/ExpressionTranslators/Internal/NpgsqlNetTopologySuiteMethodCallTranslatorPlugin.cs

```diff
@@ -1,17 +1,30 @@
-﻿using System.Linq.Expressions;
+using System.Collections.Generic;
+using System.Linq.Expressions;
+using System.Reflection;
 using GeoAPI.Geometries;
 using JetBrains.Annotations;
 using Microsoft.EntityFrameworkCore.Query.Expressions;
 using Microsoft.EntityFrameworkCore.Query.ExpressionTranslators;
 using NetTopologySuite.Geometries;
 
-namespace Npgsql.EntityFrameworkCore.PostgreSQL.NetTopologySuite
+namespace Npgsql.EntityFrameworkCore.PostgreSQL.Query.ExpressionTranslators.Internal
 {
+    public class NpgsqlNetTopologySuiteMethodCallTranslatorPlugin : IMethodCallTranslatorPlugin
+    {
+        public virtual IEnumerable<IMethodCallTranslator> Translators { get; } = new IMethodCallTranslator[]
+        {
+            new NpgsqlGeometryMethodTranslator()
+        };
+    }
+
     /// <summary>
     /// Translates methods operating on types implementing the <see cref="IGeometry"/> interface.
     /// </summary>
-    public class NetTopologySuiteMethodCallTranslator : IMethodCallTranslator
+    public class NpgsqlGeometryMethodTranslator : IMethodCallTranslator
     {
+        static readonly MethodInfo _collectionItem =
+            typeof(IGeometryCollection).GetRuntimeProperty("Item").GetMethod;
```

Comment:
I don't think this is necessary... There are many places in the translators where we assume certain types have certain methods/members, their removal would seem to be a massive breaking change, so it seems pretty safe to me...

Reply:
Let me know if you feel strongly about this.

Reply:
I've been trying to keep an eye out for this in the more recent translators so that future removals (hopefully) don't immediately trigger a bunch of maintenance work on our end.

I lean toward conditional access here, because a null `_collectionItem` just disables the translator, rather than throwing an exception.

That said, an interface's indexer should be stable, so it does seem far-fetched that this would become an issue without other big breaking changes also occuring.

Overall, I would probably still add it, but after thinking about the odds of an indexer being removed, I don't think it's as much of an issue. Either way is good with me.

Reply:
OK, will leave as-is for now. We can review all translators in a separate PR if we really feel we need to (although I think that's very low-priority).